### PR TITLE
add proper comment to doc

### PR DESCRIPTION
**Pull request by GantMan**
This exported function is missing from docs.

I'm planning on updating and helping with other functions if this PR goes through.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/1842)

### CODE REVIEW

File: tfjs-core/src/util.ts [hunk 1]
```diff
@@ -225,6 +225,18 @@ export function sizeToSquarishShape(size: number): [number, number] {
...
+ *
+ * ```js
+ * const randomTen = tf.util.createShuffledIndices(10);
+ * console.log(randomTen);
+ * ```
+ *
+ * @param number Quantity of how many shuffled indicies to create.
+ */
+/** @doc {heading: 'Util', namespace: 'util'} */
+// tslint:disable-next-line:no-any
```
Comment (nsthorat) [inline tfjs-core/src/util.ts]: you dont need this tslint

**Comment by nsthorat** [comment]
Thanks for the PR! Small comment then LGTM.

File: tfjs-core/src/util.ts (hunk 1 above)
Comment (GantMan) [reply tfjs-core/src/util.ts]: done!

**Comment by GantMan** [comment]
Woops!   removed that line.  TY!

I'll probably come back and do some more documentation, so I'll see you around :)

**Comment by GantMan** [comment]
@nsthorat - any reason the Google Cloud Build is failing?

**Pull request merged**